### PR TITLE
Add new functions

### PR DESCRIPTION
**Pull request by cbm755**

### CODE REVIEW

**Comment by cbm755** [comment]
will merge in 24 hours if no concerns

File: inst/@double/harmonic.m [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  end
+  cmd = { 'L = _ins[0]'
+          'A = [complex(harmonic(complex(x))) for x in L]'
+          'return A,' };
+  c = python_cmd (cmd, num2cell(x(:)));
+  assert (numel (c) == numel (x))
+  y = x;
+  for i = 1:numel (c)
+    y(i) = c{i};
+  end
```
Comment (latot) [inline inst/@double/harmonic.m]: maybe here use cell2sym instead?

File: inst/@sym/kroneckerDelta.m [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+function a = kroneckerDelta (n, m)
+
+  if (nargin == 1)
+    m = sym(0);
+  elseif (nargin == 2)
+    %% no-op
+  else
+    print_usage ();
+  end
```
Comment (latot) [inline inst/@sym/kroneckerDelta.m]: and here use:
```

 +  if (nargin == 1)
 +    m = sym(0);
 +  elseif (nargin > 2)
 +    print_usage ();
 +  end
```

??

File: inst/@double/harmonic.m (hunk 1 above)
Comment (cbm755) [inline inst/@double/harmonic.m]: Yes, that does look poor.  Outputs here are doubles (not sym) but I think `cell2mat` will help.

**Comment by cbm755** [comment]
thanks!  Will merge when tests pass.

**Pull request merged**